### PR TITLE
Add limit for authSessions per rootAuthSession in map storage

### PR DESCRIPTION
**Pull request by martin-kanis**

### CODE REVIEW

File: server_admin/topics/threat/auth-sessions-limit.adoc [hunk 1]
```diff
@@ -50,11 +58,37 @@ ifeval::["{kc_dist}" == "wildfly"]
 </subsystem>
 ----
 
-Equivalent configuration using CLI commands:
+The equivalent configuration for the new map storage:
```
Comment (mhajas) [inline server_admin/topics/threat/auth-sessions-limit.adoc]: I believe this is not necessary as the legacy distribution will be removed in next KC version.

**Review by mhajas — CHANGES_REQUESTED** [review]

File: server_admin/topics/threat/auth-sessions-limit.adoc (hunk 1 above)
Comment (martin-kanis) [reply server_admin/topics/threat/auth-sessions-limit.adoc]: I see. I removed parts about Wildfly.

**Review by mhajas — APPROVED** [review]
Thanks @martin-kanis

**Review by hmlnarik — APPROVED** [review]

**Comment by hmlnarik** [comment]
Fixes: keycloak/keycloak#10843

**Pull request merged**